models/utility_functions.py

```diff
@@ -430,108 +432,22 @@ def convert_act_2d_matrix(activation, kernel_y, kernel_x, stride_y, stride_x, pa
 
 
 ### Tilizing / Untilizing ###
+@deprecated("PyTorch data is handled automatically in tensor infra. This function does nothing now:")
```

Comment:
why not delete these functions?

Reply:
It's used in some stale tests. I would have to go through and remove everywhere.

Reply:
ok, not blocking for this, but the clean up is probably necessary, maybe as a separate PR.